vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManager.java

```diff
@@ -463,6 +469,44 @@ public IChoiceBlock createChoiceBlock(String slideId, List<String> selectedChoic
         return choiceBlockRepo.save((ChoiceBlock) choiceBlock);
     }
 
+    @Override
+    public IBiblioBlock createBiblioBlock(String slideId, String title, String description, Integer contentOrder) {
+        ISlide slide = slideManager.getSlide(slideId);
+        IBiblioBlock bilioBlock = new BiblioBlock();
+        bilioBlock.setDescription(title);
+        bilioBlock.setBiblioTitle(description);
+        bilioBlock.setSlide(slide);
+        bilioBlock.setContentOrder(contentOrder);
+        return biblioBlockRepo.save((BiblioBlock) bilioBlock);
+    }
+
+    @Override
+    public void deleteBiblioBlockById(String id) throws BlockDoesNotExistException {
+        if (id == null) {
+            return;
+        }
+        
+        try {
+            biblioBlockRepo.deleteById(id);         
+        } catch (IllegalArgumentException e) {
+            System.err.println("Failed to delete biblio block: " + e.getMessage());
```

Comment:
This is not how exceptions should be handled. Ever. I'm fairly certain we had a bi-weekly about this.

Exceptions need to either:
- be handled (logically that is, meaning is there a direct consequence that should happen based on the exception? Or can we just log and return? Log and ignore? note that *log* here means actual logging, system out is not logging, since you never know where those messages end up (if they end up somewhere at all))
- thrown up to the next level for them to handle. If that's the case then is it a general enough exception that can be thrown up without causing tight coupling? if so then just add a `throws` with the type of exception and that's it. If it would cause tight coupling, then wrap the exception in another exception of a type that makes sense to through here (e.g. `BlockDoesNotExistException` which this method throws according to its signature).

Reply:
I didn't write this piece of code but I'll make changes to this, by adding logger, let me know if that is good enough